src/style/themes/large/large-theme.config.js

```diff
@@ -9,7 +9,10 @@ export default (palette) => {
         base: palette.amethyst,
         primary: palette.amethystTint(10),
         secondary: palette.amethystShade(10),
-        tertiary: palette.amethystShade(30)
+        tertiary: palette.amethystShade(30),
+
+        // status
+        focus: palette.goldTint(50)
```

Comment:
Why have you added this here? What is the purpose of this colour?

Reply:
This is on design of the DLS